docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -80,6 +80,10 @@ execution on a Trino cluster:
   - Enable compression of spooling data. Setting to `true` is recommended
     when using an [exchange manager](fte-exchange-manager).
   - ``false``
+  - `fault-tolerant-execution.exchange-encryption-enabled`
+  - Encrypt data before spooling, see [Encryption](fte-encryption) for details. 
+    Setting this property to false is not recommended.
```

Comment:
Why? Is there no performance overhead at all  thats worth thinking about?

Reply:
Because it is less secure. 
* `fault-tolerant-execution.exchange-encryption-enabled = true` = secure
* `fault-tolerant-execution.exchange-encryption-enabled = true` + TLS = secure, but wastes CPU
* `fault-tolerant-execution.exchange-encryption-enabled = false` + TLS = insecure. Anyone with access to storage can read the data
* `fault-tolerant-execution.exchange-encryption-enabled = false` = totally insecure

Reply:
Sure but the secure option has a performance overhead right? and if the storage and network are totally trusted there is nothing to gain...

Reply:
I added a qualifier - this could be disabled if the data is not sensitive.